source/components/molecules/Modal/Modal.js

```diff
@@ -1,33 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import RnModal from 'react-native-modal';
-import styled from 'styled-components/native';
-import { SafeAreaView } from 'react-native-safe-area-context';
+import { Modal as RnModal } from 'react-native';
```

Comment:
prefers to print the full name. more clearly.
`ReactNativeModal`

Reply:
Fixed